map/games/pact_of_steel_2.xml

```diff
@@ -4011,10 +4014,10 @@
       <option name="activateTrigger" value="triggerAttachment_Americans_Aid_UK:1:false:false:false:false"/>
       <option name="text" value="AMERICANS_AID_UK"/>
       <option name="chance" value="6:6"/>
+      <option name="costResources" value="PUs" count="10"/>
+      <!--<option name="costResources" value="Fuel" count="5"/>-->
```

Comment:
Just download it from my repositories and you can test that it does.

`<option name="costResources" value="PUs" count="10"/>`
and
`<option name="costResources" value="10:PUs"/>`
are two different ways to define the same thing.

Personally, I definitely believe spelling out the count is preferable, for a series of reasons (logically descriptive, more telling, easier to read, much easier to work with for mass changes, self-telling without needing knowing how the specific option works (you can have options accepting multiple values set apart by colons, and some of those may, then, have a count, like for a minimum needed)). I would never use the value-only format (I dislike it even exists as an option), but I kept it for the other cases, guessing was your preference.

For example, you can do this mass change (either ways) (this is what I already did for the maps I'm going to update):
*before|after*
*name="costPU" value=|name="costResources" value="PUs" count=*
but you cannot mass change to the value only format in a normal way.

Anyways, whatever the personal preference, both work and are supported.

Reply:
Are you positive about that? Did you test it? I thought that count essentially appended that field to value so that it would end up being "PUs:10" instead of "10:PUs".

Reply:
Yeah, that is another reason to go for the easier to learn version with the count. I've seen a few cases of mapmakers that would get confused and write the count after the value. Anyways, in such cases, if you don't have the count, the format is that the first item of the value is the count.

I certainly tested it (or rather verified it), with the latest prerelease.

Really, that is needed just for the Triggers that change options' values, but this is not relevant here.

Reply:
Anyways, I confirm that the version with the count spelled out gets turned into the same thing with the count at start of the value. My preference for the longer formula with the count is based purely on what I said as an xml maker / reader.

Reply:
Meaning I prefer to write it down that way in pos2 for a bigger chance of, then, mapmakers coding it that way, in their xml, that I think is easier to read and work with and mass change. This is the reason why I changed the format, but of course, you can use either.

Actually, I'd like to change to that format also the commented out options for the political actions examples.